### PR TITLE
Remove sqlite package after it did its job

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,6 +10,8 @@ RUN dnf install -y python3 python3-requests && \
 RUN mkdir -p /var/lib/ironic-inspector && \
     sqlite3 /var/lib/ironic-inspector/ironic-inspector.db "pragma journal_mode=wal"
 
+RUN dnf remove -y sqlite
```
Comment (dtantsur) [inline Dockerfile]: nit: maybe merge into the previous `RUN` to reduce the number of layers?

**Review by dtantsur — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (elfosardo) [reply Dockerfile]: yeah.... I completely ignored that for some reason

**Review by dtantsur — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -8,7 +8,8 @@ RUN dnf install -y python3 python3-requests && \
     rm -rf /var/cache/{yum,dnf}/*
 
 RUN mkdir -p /var/lib/ironic-inspector && \
-    sqlite3 /var/lib/ironic-inspector/ironic-inspector.db "pragma journal_mode=wal"
+    sqlite3 /var/lib/ironic-inspector/ironic-inspector.db "pragma journal_mode=wal" && \
+    dnf remove -y sqlite
```
Comment (maelk) [inline Dockerfile]: Should the dnf clean commands also be called after this operation ? to remove the cache for example, or is it not needed ?

Comment (elfosardo) [reply Dockerfile]: thanks for the suggestion
I think I will just compress the two RUN commands in a single one and put the cache clean commands at the very end

**Pull request merged**